src/Language/AST/Node.php

```diff
@@ -39,6 +39,9 @@ abstract class Node
     /** @var Location */
     public $loc;
 
+    /** @var string */
+    public $kind;
```

Comment:
This will be very useful for other libraries using graphql-php, see https://github.com/rebing/graphql-laravel/blob/a458a79f7bad6406f181ac4d1ab371d65aa12c7f/phpstan.neon#L37